pkg/console/operator/status.go

```diff
@@ -136,70 +141,27 @@ func (c *consoleOperator) ConditionNotFailing(operatorConfig *operatorsv1.Consol
 	return operatorConfig
 }
 
-func (c *consoleOperator) ConditionProgressing(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeProgressing,
-		Status:             operatorsv1.ConditionTrue,
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-func (c *consoleOperator) ConditionNotProgressing(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeProgressing,
-		Status:             operatorsv1.ConditionFalse,
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
-func (c *consoleOperator) ConditionAvailable(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeAvailable,
-		Status:             operatorsv1.ConditionTrue,
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
-func (c *consoleOperator) ConditionNotAvailable(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeAvailable,
-		Status:             operatorsv1.ConditionFalse,
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
 // When a sync failure happens,
-// we dont know if the operand is available
+// we dont know if the operand is available, so it is best not to set it.
+//  on install, an incomplete sync will mean the operator is unavailable.
+//  however, on a later sync when a change is encountered, this is not true.
 // we do know we are progressing because we are trying to change something about the operand
 // we do know we failed to make the update
-func (c *consoleOperator) ConditionResourceSyncFailure(operatorConfig *operatorsv1.Console, message string) *operatorsv1.Console {
-	// message := "The operator failed to update a resource of the operand."
```

Comment:
`SyncFailure` is not exactly accurate, this is used to imply we deliberately aborted the sync loop with an early return, with the intent that the next sync loop will pick up and progress our state further.  Therefore, the name change to simply `SyncIncomplete` and removal of a statement on `Availability`.